@here/harp-mapview/lib/MapView.ts

```diff
@@ -1984,13 +1988,25 @@ export class MapView extends THREE.EventDispatcher {
         this.dispatchEvent(DID_RENDER_EVENT);
     }
 
-    private renderTileObjects(tile: Tile, zoomLevel: number) {
+    // Helper method to extract the list of objects from the supplied Tile or TileProxy
+    private getObjects(tileOrProxy: Tile | TileProxy) {
+        return tileOrProxy instanceof Tile ? tileOrProxy.objects : tileOrProxy.clonedObjects;
```

Comment:
if you rename "clonedObjects" to "objects", this whole function would not be required.

Reply:
Agreed